src/main/java/com/codeborne/selenide/hookactions/HookAction.java

```diff
@@ -0,0 +1,25 @@
+package com.codeborne.selenide.hookactions;
+
+import org.openqa.selenium.WebElement;
+
+public interface HookAction {
+  /**
+   * Condition for activation
+   *
+   * @param element     WebElement
+   * @param methodName  Name of method, who called hook
+   * @param args        Called command arguments
+   * @return boolean
+   *
+   */
+  boolean conditionForAction(WebElement element, String methodName, Object... args);
```

Comment:
Maybe instead of conditionForAction => isEnabled() or isActive() will be better?

Reply:
fixed in last commit